docs/ff-concepts/adding-customization/common-examples.md

```diff
@@ -0,0 +1,284 @@
+---
+slug: /concepts/custom-code/common-examples
+title: Common Examples
+description: Learn about the common custom code examples and use it directly in your project.
+tags: [Custom Actions, Custom Code]
+sidebar_position: 6
+keywords: [FlutterFlow, Custom Actions, Customizations, Flutter, Dart, Pub.dev, Examples]
+---
+
+# Common Code Examples
```

Comment:
Probably want to call out what can be used in custom functions vs actions - like I dont think you can use app state in functions?